### PR TITLE
#162: Ensure Chrome Custom Tab does not disappear…

### PR DESCRIPTION
**Pull request by tp**
…when switching away and back from the app during Sign in with Apple

### CODE REVIEW

File: packages/sign_in_with_apple/sign_in_with_apple/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt [hunk 1]
```diff
@@ -91,7 +91,6 @@ public class SignInWithApplePlugin: FlutterPlugin, MethodCallHandler, ActivityAw
 
         val builder = CustomTabsIntent.Builder();
         val customTabsIntent = builder.build();
-        customTabsIntent.intent.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
```
Comment (HenriBeck) [inline packages/sign_in_with_apple/sign_in_with_apple/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt:94]: So the fix is to keep the activity on the history?

I can't exactly recall why we added this in the initial version, do you?

Comment (tp) [reply packages/sign_in_with_apple/sign_in_with_apple/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt:94]: Tested both deep links (SiwA and normal app (main activity) ones), and they would with both `launchMode`s close the CCT. For the in-app deep link the deep link handler would be called inside the app. 

So this seems fine as far as I can tell.

File: packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+### 5.0.0
```
Comment (tp) [inline packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md:1]: After the extensive testing, I wouldn't deem this a breaking change, as in both cases (`launchMode`s) it just works better.

So if testing from the reports in #162 is also fine, I would consider releasing this as just `4.3.0`.

What do you think @HenriBeck ?

File: packages/sign_in_with_apple/sign_in_with_apple/example/README.md [hunk 3]
```diff
@@ -13,3 +13,25 @@ flutter config --enable-macos-desktop
 # cd macos && bundle exec pod install && cd -
 flutter run -d macOS
 ```
+
+## Test Android deep links
```
Comment (tp) [inline packages/sign_in_with_apple/sign_in_with_apple/example/README.md:17]: Extended the sample app a bit so we can easily test the deep links with it.

File: packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md (hunk 2 above)
Comment (HenriBeck) [reply packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md:1]: Should we add the docs also the the README?

**Review by HenriBeck — APPROVED** [review]

File: packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md (hunk 2 above)
Comment (tp) [reply packages/sign_in_with_apple/sign_in_with_apple/CHANGELOG.md:1]: Good point. I'll adapt the version number and add the `launchMode` info into the README for Android.

**Pull request merged**